src/core/components/ChatHistory.tsx

```diff
@@ -37,25 +40,16 @@ export const ChatHistory = () => {
 
   return (
     <div className={styles.chatHistoryContainer}>
-      <button
-        onClick={() => messageHistoryStore.reset()}
-        className={styles.newChatButton}
-        data-testid="new-chat-container-button"
-      >
-        <svg
```

Comment:
This diff formatted funny. All I did here was take out the <svg> code and replaced it with the exact same SVG code as a component: NewChatIcon since that will need to be used in multiple places now